test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/K8SequentialExecutor.java

```diff
@@ -75,6 +83,23 @@
 
     }
 
+    private Map<String, V1ContainerStatus> getPravegaPodStatus(K8sClient client) {
+        return Futures.getAndHandleExceptions(
+                client.getRestartedPods("default", "app", "pravega-cluster"),
```

Comment:
We need to be sure of what we need to include to check for restarts. I agree that a Controller and Segment Store restart should be interpreted as a failure. This may be also extended to unexpected ZK and BK pod restarts. However, we need to make sure that we do not include any of the other pods available by default in the cluster (e.g., fluentd).

Reply:
At present with the label `app=pravega-cluster` tracks the restart counts for bookies, Segment store and Controller.

Reply:
Updated the code to track zk pod status .